glances/outputs/static/html/stats.html

```diff
@@ -29,6 +29,9 @@
         <div class="col-sm-12 col-md-8 col-lg-6" ng-if="!arguments.disable_cpu && arguments.percpu">
             <section id="per_cpu" class="plugin" ng-include src="'plugins/per_cpu.html'"></section>
         </div>
+        <div class="hidden-xs hidden-sm hidden-md col-lg-3" ng-if="!arguments.disable_gpu && statsGpu.gpus.length > 0">
```

Comment:
The easier way that I see to have enough space for this plugin is to hide the mem_more part when this plugin is displayed. You can do this by using the following code for the ng-if of mem_more : 

`!arguments.disable_mem && statsGpu.gpus.length = 0`

If you are ok with solution, you need to display the gpu plugin in medium(md) size too:

`hidden-xs hidden-sm col-md-4 col-lg-3`